### PR TITLE
add in-flight icing products to inline post

### PR DESCRIPTION
**Pull request by SamuelTrahanNOAA**
## Description

Add in-flight icing products to the inline post.

A full description of the new feature, and status of the draft, can be found in this merged UPP PR:

https://github.com/NOAA-EMC/UPP/pull/536

### Issue(s) addressed

https://github.com/ufs-community/ufs-weather-model/issues/1647

## Testing

How were these changes tested?  Results for unaffected tests do not change. New fields are created with valid content, unless library is absent, then the fields have missing values.

What compilers / HPCs was it tested with?  Tests were run on hera and jet, but the target machine is WCOSS2; the library has restricted access and won't be available elsewhere.

Are the changes covered by regression tests?  Yes.

## Dependencies

https://github.com/NOAA-EMC/UPP/pull/655

### CODE REVIEW

**Comment by DusanJovic-NOAA** [comment]
@SamuelTrahanNOAA What's the status of this PR? I hasn't been updated for 3 months.

**Comment by SamuelTrahanNOAA** [comment]
We're waiting for permission to install libIFI on WCOSS2. The IP situation is complicated, and management is being slow.

**Comment by SamuelTrahanNOAA** [comment]
This PR is scheduled to be merged soon and nobody has reviewed it.

File: io/post_nems_routines.F90~
Comment (WenMeng-NOAA) [inline io/post_nems_routines.F90~:1]: @SamuelTrahanNOAA I would think you inadvertently committed this file?

Comment (SamuelTrahanNOAA) [reply io/post_nems_routines.F90~:1]: It is gone now.

**Review by DusanJovic-NOAA — APPROVED** [review]

**Review by WenMeng-NOAA — APPROVED** [review]

**Comment by SamuelTrahanNOAA** [comment]
This was merged as part of #628

**Pull request closed without merging**